src/config/drive.rs

```diff
@@ -22,17 +30,17 @@ pub struct Drive<'d> {
 
 impl<'d> Drive<'d> {
     /// Create a new `DriveBuilder` instance.
-    pub fn builder<I, P>(drive_id: I, path_on_host: P) -> DriveBuilder<'d>
+    pub fn builder<I>(drive_id: I) -> DriveBuilder<'d>
     where
         I: Into<Cow<'d, str>>,
-        P: Into<Cow<'d, Path>>,
     {
         DriveBuilder(Drive {
             drive_id: drive_id.into(),
             is_read_only: false,
             is_root_device: false,
             part_uuid: None,
-            path_on_host: path_on_host.into(),
+            path: Path::new("").into(),
+            path_on_host: Path::new("").into(),
```

Comment:
At least one of them is a must for user to provide, which is why it was a parameter to `builder`.

Reply:
yeah fixed